coredb-operator/src/defaults.rs

```diff
@@ -6,6 +6,10 @@ pub fn default_postgres_exporter_enabled() -> bool {
     true
 }
 
+pub fn default_service_monitor_enabled() -> bool {
+    false
+}
```

Comment:
This feature is default off because the user has to have already installed the ServiceMonitor CRD for it to work.